contracts/proxy/utils/UUPSUpgradeable.sol

```diff
@@ -25,7 +25,7 @@ abstract contract UUPSUpgradeable is IERC1822Proxiable {
      * and `upgradeToAndCall(address,bytes)` are present, and `upgradeTo` must be used if no function should be called,
      * while `upgradeToAndCall` will invoke the `receive` function if the second argument is the empty byte string.
      * If the getter returns `"5.0.0"`, only `upgradeToAndCall(address,bytes)` is present, and the second argument must
-     * be the empty byte string if no function should be called, being impossible to invoke the `receive` function
+     * be the empty byte string if no function should be called, making impossible to invoke the `receive` function
```

Comment:
```suggestion
     * be the empty byte string if no function should be called, making it impossible to invoke the `receive` function
```

Reply:
Fixed in https://github.com/OpenZeppelin/openzeppelin-contracts/commit/68963df79ea364f3ddf8bcd9e70aab119dac45ab